modules/utxo-lib/src/bitgo/wallet/Unspent.ts

```diff
@@ -39,6 +39,11 @@ import { ProprietaryKeySubtype, PSBT_PROPRIETARY_IDENTIFIER } from '../PsbtUtil'
 /** Final (non-replaceable) */
 export const TX_INPUT_SEQUENCE_NUMBER_FINAL = 0xffffffff;
 
+/** Non-Final (Replaceable)
+ * Reference: https://github.com/bitcoin/bitcoin/blob/88c3b100f0e83318b9379b17ba7bd2a1c5a51b74/src/rpc/rawtransaction_util.cpp#L49
+ * */
+export const TX_INPUT_SEQUENCE_NUMBER_NON_FINAL = 0xffffffff - 2;
```

Comment:
```suggestion
export const MAX_BIP125_RBF_SEQUENCE = 0xffffffff - 2;
```

(more precise, there are many nonfinal numbers but only one max bip125 rbf sequence number)